packages/backend/src/models/ValidationModel/ValidationModel.ts

```diff
@@ -104,7 +104,9 @@ export class ValidationModel {
                 projectUuid: validationError.project_uuid,
                 error: validationError.error,
                 name: validationError.name,
-                lastUpdatedBy: `${validationError.first_name} ${validationError.last_name}`,
+                ...(!!validationError.first_name && {
```

Comment:
Not sure I prefer this syntax, or 
```
lastUpdatedBy: validationError.first_name && `${validationError.first_name} ${validationError.last_name}` 
```

or even 

```
lastUpdatedBy: validationError.first_name  ? `${validationError.first_name} ${validationError.last_name}` : undefined
```

We don't need to completely remove the property, just set it to undefined.

Reply:
sounds good, open to both, will change !